src/UNL/Peoplefinder/Driver/LDAP/StandardFilter.php

```diff
@@ -48,9 +48,13 @@ function __construct($inquery, $operator = '&', $wild = false)
 
             //escape query
             $inquery = UNL_Peoplefinder_Driver_LDAP_Util::escape_filter_value($inquery);
-            
+
             //put the query into an array of words
             $query = preg_split('/\s+/', $inquery, 4);
+            if (count($query) > 1) {
+                //add the original multi-word query as a search term
+                $query[] = $inquery;
```

Comment:
That's true. I did some tests of the LDAP filter before and after this change, and the number of results for my tests were the same, but I'm having trouble proving it will always be the same, or if any differences will matter. Take a look at this, the inquery/as_entered was always added and wrapped with an OR `|` operator. See the old line 92: https://github.com/unl/UNL_Directory/pull/35/files#diff-c4a2381d4542bdbc325ada2dfb54cf98L92

and here's a screenshot of the old vs new for `('brett bieber','&',false)`:
![screen shot 2016-02-22 at 4 20 47 pm](https://cloud.githubusercontent.com/assets/92284/13234853/6e5b093c-d980-11e5-9815-d0d5ddffa11d.png)